source/modules/assets/global.rst

```diff
@@ -0,0 +1,13 @@
+Global search
+=============
+
+Menu `Asset > Global` allows to perform a global search on all allowed elements. This search is limited to following assets:
+
+* computer
```

Comment:
Capitalize item types.
There are a lot more types included in 9.5's global search.
See $CFG_GLPI["globalsearch_types"] in inc/define.php